index.js

```diff
@@ -1,186 +1,220 @@
+      context.log.debug(
+        `Not responding to event from non-Travis app (slug=${slug})`
+      );
+      return;
     }
 
-    const repoDir = await github.downloadRepo(repoUrl)
-    const config = await loadConfig(context)
+    const repoDir = await github.downloadRepo(repoUrl);
+    const config = await loadConfig(context);
 
-    const travisBuildId = travis.getBuildIdFromDetailsUrl(detailsUrl)
-    const travisBuild = await travis.getBuildFromId(travisBuildId)
+    const travisBuildId = travis.getBuildIdFromDetailsUrl(detailsUrl);
+    const travisBuild = await travis.getBuildFromId(travisBuildId);
 
-    logImportantInformation(context, travisBuild)
+    logImportantInformation(context, travisBuild);
 
-    const shouldPrune = shouldPruneRedundantFeatures(context, config, travisBuildId)
-    const shouldMerge = shouldMergeAcceptedFeature(context, config, travisBuild)
-    const shouldClose = shouldCloseRejectedFeature(context, config, travisBuild)
+    const shouldPrune = shouldPruneRedundantFeatures(
+      context,
+      config,
+      travisBuildId
+    );
+    const shouldMerge = shouldMergeAcceptedFeature(
+      context,
+      config,
+      travisBuild
+    );
+    const shouldClose = shouldCloseRejectedFeature(
+      context,
+      config,
+      travisBuild
+    );
 
-    const shouldPruneResult = await shouldPrune
+    const shouldPruneResult = await shouldPrune;
     if (shouldPruneResult.result) {
-      context.log.info('Pruning features...')
-      await pruneRedundantFeatures(context, repoDir.name, config, travisBuild)
+      context.log.info("Pruning features...");
+      await pruneRedundantFeatures(context, repoDir.name, config, travisBuild);
     } else {
-      context.log.info(`Not acting to prune features because ${shouldPruneResult.reason}`)
+      context.log.info(
+        `Not acting to prune features because ${shouldPruneResult.reason}`
+      );
     }
 
-    const shouldMergeResult = await shouldMerge
-    if (shouldMergeResult.result) {
-      context.log.info('Merging PR...')
+    const shouldMergeResult = await shouldMerge;
+    if (shouldMergeResult.shouldMerge) {
+      context.log.info("Merging PR...");
+      const { omittedFeature } = travis.getTravisAcceptanceMetadata(buildId);
+      const omittedFeatureMessage = omittedFeature
+        ? `We found that your feature provided more information than another feature: ${omittedFeature}`
+        : `We found that your feature added valuable information on top of all the existing features`;
       const message = dedent`
-        After validation, your feature was accepted. It will be automatically merged into the project.
-      `
-      await github.commentOnPullRequest(context, travisBuild.pull_request_number, message)
-      await github.mergePullRequest(context, travisBuild.pull_request_number)
-      await github.closePullRequest(context, travisBuild.pull_request_number)
+        After validation, your feature was accepted.
```

Comment:
Can this be a single paragraph with no newlines? It will render better on GitHub.